test/quantum_mechanics/harmonic_oscillator/quantum_oscillator_wave_eigenfunctions_test.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy import sqrt
+from symplyphysics import (
+    assert_equal,
+    units,
+    Quantity,
+    errors,
+)
+from symplyphysics.core.symbols.quantities import evaluate_quantity
+from symplyphysics.laws.quantum_mechanics.harmonic_oscillator import wave_eigenfunctions as wave_law
+
+Args = namedtuple("Args", "n m w x")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    n = 1
+    m = evaluate_quantity(units.planck_mass)
```

Comment:
Add comment, why you are doing it. Otherwise someone could decide to refactor it in future.

Reply:
Sure